server/main.js

```diff
@@ -42,6 +43,15 @@ if (config.env === 'development') {
   app.use(express.static(publicPath))
   app.use('/static', express.static('server/static'))
 
+  // Most probably the following requests should be answered by the Nice2 instance
+  // -> pipe them through
+  app.use(['/nice2/*', '/js/*', '/img/*'], function(req, res, next) {
+    const newUrl = 'http://localhost:8080' + req.originalUrl
```

Comment:
Maybe use __BACKEND_URL__ instead of hardcoded 8080. Then it should work with master backend as well